src/public/style.css

```diff
@@ -0,0 +1,118 @@
+/* Opinionated CSS defaults based on:

```

Comment:
The following code is a reset file which was added after running _npm create-javascript-project_. I needed a reset file anyway, so this worked out nicely!